Domain/sdLitica.Events/Bus/EventBus.cs

```diff
@@ -0,0 +1,50 @@
+﻿using sdLitica.Events.Abstractions;
+using sdLitica.Events.Extensions;
+using sdLitica.Messages.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace sdLitica.Events.Bus
+{
+    public class EventBus : IEventBus
+    {
+        private readonly IPublisher _publisher;
+        private readonly IConsumer _consumer;
+        private readonly IEventRegistry _eventRegistry;
+
+        public EventBus(IPublisher publisher, IConsumer consumer, IEventRegistry eventRegistry)
+        {
+            _publisher = publisher;
+            _consumer = consumer;
+            _eventRegistry = eventRegistry;
+        }
+
+        public void Publish(IEvent @event)
+        {
+            var message = @event.ToMessage();
```

Comment:
I know that C# allows us to use var instead of type, but I would recommend using explicit type, this makes code much easier to read and work with it. As a team we need to agree on code-style and if everybody else prefer var - I'm ok, but my personal preference is using explicit types

Reply:
Great. No problems.